src/vaadin-usage-statistics.js

```diff
@@ -342,7 +342,8 @@ class UsageStatistics {
 }
 
 try {
-  new UsageStatistics().maybeGatherAndSend();
+  window.Vaadin.usageStatistics = window.Vaadin.usageStatistics || new UsageStatistics();
```

Comment:
The rest of the code guards against `window.Vaadin` not being defined so should probably do that here also

Reply:
Done.